test/default-downloads-test.js

```diff
@@ -184,27 +184,6 @@ describe('default-downloads', function() {
   });
 
   describe('firefox', function() {
-    describe('arm', function() {
```

Comment:
Starting from `0.24.0` release, arm builds are not supported.

Reply:
If arm builds were supported previously, how big do you think it's an issue that they won't be supported now? Do you have any link to the source information on that?

Thanks for the PR!

Reply:
There are more details about arm build removal: https://github.com/mozilla/geckodriver/issues/1515

> how big do you think it's an issue 

Also, there were never arm builds for Chromedriver, so I am not sure if there were active users of this setup.

If you think, backward-compatibility is important here, we could have a special condition for arm to download the last available version.